jazz_cloud-logs-streamer/components/utils.js

```diff
@@ -0,0 +1,83 @@
+/**
+    Helper functions for Cloud Logs Streamer
+  @module: utils.js
+  @description: Helper functions for using Regex patterns, etc.
+  @author: SIDD
+  @version: 1.0
+**/
+
+const crypto = require('crypto');
+const logger = require("../components/logger.js"); 
+
+logger.setLevel('debug');
+
+// Helper functions
+
+var extractJson = function(message) {
+    var jsonStart = message.indexOf('{');
+    if (jsonStart < 0) return null;
+    var jsonSubString = message.substring(jsonStart);
+    return isValidJson(jsonSubString) ? jsonSubString : null;
+}
+
+var isNumeric = function(n) {
+    return !isNaN(parseFloat(n)) && isFinite(n);
+}
+
+var isValidJson = function(message) {
+    try {
+        JSON.parse(message);
+    } catch (e) { return false; }
+    return true;
+}
+
+
+var getInfo = function(messages, patternStr) {
+	var pattern = new RegExp(patternStr);
```

Comment:
fix indentation

Reply:
Indentation and formatting fixed and files